ci-automation/ci-config.env

```diff
@@ -53,3 +53,8 @@ QEMU_BIOS="/usr/share/qemu/bios-256k.bin"
 # Published by vms.sh as part of the qemu vendor build.
 QEMU_UEFI_BIOS="flatcar_production_qemu_uefi_efi_code.fd"
 
+# -- VMware ESX --
+
+ESX_IMAGE_NAME='flatcar_production_vmware_ova.ova'
+ESX_PARALLEL="${PARALLEL_TESTS:-4}"
```

Comment:
with the other PR to set up the creds we now have two places where it's defined, I think since the default here depends on the address setup we could also use `1` here or error out if not defined

Reply:
Now it is expected that the creds will come through the sdk_container/.env or some other way.